src/webots/app/WbApplication.hpp

```diff
@@ -20,6 +20,7 @@
 // instanciated/deleted by either WbGuiApplication or WbConsoleApplication
 //
 
+#include <QtCore/QElapsedTimer>
```

Comment:
The definition is not needed here, we should simply declare the class:
```
class QElapsedTimer;
```

and remove the QTime class declaration.